cli/uiUtils/confirmation.go

```diff
@@ -15,7 +16,7 @@ func AskForConfirmation(s string) bool {
 
 	response, err := reader.ReadString('\n')
 	if err != nil {
-		log.Fatal(err)
+		logger.Log.Errorf(err.Error())
```

Comment:
I think it needs to be changed to fatal, and maybe add informative message like "Error while reading confirmation string, err: %v"